roles/gdm/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Install GNOME Display Manager (GDM)
+  pacman: name=gdm state=present
+
+- name: Enable GDM
+  service: name=gdm.service enabled=yes
+
+- name: Creating DConf configuratioin profile
+  copy: src=profile/gdm dest=/etc/dconf/profile/gdm
+
+- name: Creating gdm.d directory for DConf
+  file: path=/etc/dconf/db/gdm.d state=directory owner=root group=root mode=0644
+  when: gdm.logo_path or
```

Comment:
This is fine the way you have it, but since an empty `/etc/dconf/db/gdm.d` won't hurt anything, I'd probably remove all the conditionals and just always create the directory. That makes the task a bit simpler to maintain.